src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -209,6 +218,17 @@ static ConfigDef newConfigDef() {
             5,
             ConfigDef.Width.NONE,
             SNOWFLAKE_SCHEMA)
+        .define(
+            SNOWFLAKE_ROLE,
+            Type.STRING,
+            null,
+            nonEmptyStringValidator,
+            Importance.LOW,
+            "Snowflake user name",
```

Comment:
Update comment to be clear it's a role and not a user

Reply:
Oops, thanks for the catch.